atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfig.java

```diff
@@ -223,6 +223,15 @@ public int getDefaultLockTimeoutSeconds() {
         return AtlasDbConstants.DEFAULT_LOCK_TIMEOUT_SECONDS;
     }
 
+    /**
+     * Returns a configuration for this timestamp client, indicating whether timestamp batching
+     * should be used, and if so, the minimum time interval between batches.
+     */
+    @Value.Default
+    public TimestampClientConfig timestampClient() {
```

Comment:
How hard would it be to make this live-reloadable (i.e. part of the upcoming `RuntimeConfig`)?